20220511140915_create_preset_category_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class CreatePresetCategoryTable extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * Write your reversible migrations using this method.
+     *
+     * More information on writing migrations is available here:
+     * https://book.cakephp.org/phinx/0/en/migrations.html#the-change-method
+     *
+     * Remember to call "create()" or "update()" and NOT "save()" when working
+     * with the Table class.
+     */
+    public function change(): void
+    {
+        $table = $this->table('preset_categories');
+        $table
+            ->addColumn('name', 'string', ['limit' => 64, 'null' => false])
+            ->addColumn('enabled','boolean', ['default' => false, 'null' => false])
+            ->addColumn('icon',"string",["null"=>true])
```

Comment:
I don't think we need the icon name here. It should be guessable from the category name. Convention over configuration.